webapp/src/store/events.tsx

```diff
@@ -20,7 +21,7 @@ export const createEvent = async (event: Event) => {
 }
 
 export const saveEvent = async (event: Event) => {
-  calcSlotTimes(event);
+  event = calcSlotTimes(event);
```

Comment:
Double check this for me to make sure it's reasonable - I did this to avoid end_datetime from being discarded in `docToEvent`.

Reply:
I think this is perfect!